app/src/main/res/layout/home_fragment.xml

```diff
@@ -4,48 +4,72 @@
+        android:orientation="vertical"
         android:padding="@dimen/home_fragment_padding">
 
         <ImageView
             android:id="@+id/img_device_status"
             android:layout_width="@dimen/device_status_width"
             android:layout_height="@dimen/device_status_height"
             android:layout_gravity="center"
-            android:src="@drawable/usb_disconnected" />
+            android:src="@drawable/icons_usb_disconnected_100" />
 
         <TextView
             android:id="@+id/tv_device_status"
             android:layout_width="match_parent"
             android:layout_height="@dimen/device_status_height"
             android:gravity="center"
             android:padding="@dimen/home_fragment_padding"
-            android:text="@string/pslab_device_not_found"
+            android:text="@string/device_status_text"
+            android:textColor="@color/blue_grey"
             android:textSize="@dimen/device_status_textsize"
             android:textStyle="bold" />
-    </LinearLayout>
 
-    <TextView
-        android:id="@+id/tv_device_version"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/home_fragment_textview_margin"
-        android:gravity="center"
-        android:padding="@dimen/home_fragment_padding"
-        android:textSize="@dimen/device_status_textsize"
-        android:textStyle="bold" />
+        <TextView
+            android:id="@+id/tv_connect_msg"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/device_status_height"
+            android:gravity="top|center"
+            android:paddingLeft="@dimen/padding_20dp"
+            android:paddingRight="@dimen/padding_20dp"
+            android:text="@string/pslab_connect_instruction_text"
+            android:textColor="@color/blue_grey"
+            android:textSize="@dimen/device_status_textsize"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/tv_device_version"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/device_status_height"
+            android:gravity="top|center"
+            android:paddingLeft="@dimen/padding_20dp"
+            android:paddingRight="@dimen/padding_20dp"
+            android:textColor="@color/blue_grey"
+            android:textSize="@dimen/device_status_textsize"
+            android:textStyle="bold"
+            android:visibility="gone" />
+    </LinearLayout>
 
     <TextView
-        android:id="@+id/tv_initialisation_status"
-        android:layout_width="match_parent"
+        android:id="@+id/tv_device_description"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/home_fragment_textview_margin"
-        android:gravity="center"
+        android:layout_gravity="center"
+        android:layout_marginTop="30dp"
```

Comment:
@Avjeet you have missed this one. And also adding dimensions in the `dimens.xml` files, don't put IDs like `padding_20dp` etc.

The idea is to have the resources placed in a globally available file where another developer can easily change the dimensions or texts. In that sense `padding_20dp` will have a better meaning if it becomes `home_textview_padding`.

Reply:
ok 👍